steering_committee_guidelines.rst

```diff
@@ -0,0 +1,87 @@
+
+Expectations of a Committee Member:
+===================================
+
+As a steering committee member, you agree to:
+
+1. Abide by the Ansible Community Code of Conduct.
+2. Join and participate in weekly meetings and asynchronous discussions.
+3. Not promote corporate or individual agenda during the decision making
+   process.
+4. Review other proposals and provide feedback.
+5. Represent the needs of the community.
+6. Engage with the community in a professional and positive manner.
+7. Encourage community members to express their opinion.
+
+Joining the Steering Committee:
+===============================
+
+Eligibility:
+~~~~~~~~~~~~
+
+1. Have a wide knowledge of Ansible and its related projects.
+2. Active Contributions to the Ansible project.
+3. Voicing out opinions whenever necessary.
+
+Process:
+~~~~~~~~
+
+1. Any community member may nominate someone/themselves for committee
+   membership at any time.
+2. Only the present Steering committee members may vote on selecting a
+   new member.
+3. Nominees must receive a majority vote from the present members to be
+   added to the Steering Committee.
+
+Leaving the Steering Committee:
+===============================
+
+Steering Committee members can resign voluntarily or be removed by the
+rest of the Steering Committee under certain circumstances, see details
+below.
+
+Voluntarily Leaving the Steering Committee:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A Steering Committee member can resign. If they are not able to
+participate fully for any reason, they can notify the other members and
+step down.
+
+Steering Committee members who resign and later become active again can
+rejoin the Steering Committee by following the process for joining the Steering Committee.
+
+In case of resignation, the member should create an issue in the
+community-topics repository to inform the community that they plan to
+step down and when.
+
+Involuntarily Leaving the Steering Committee:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A Steering Committee member will be removed from the Committee if they:
+
+1. Do not participate in community meetings for more than 3 months in a row.
+2. Participate unreasonably irregularly (for example, once a month for several months). Unreasonability is defined by other steering committee members considering circumstances in each particular case.
+3. Violate the Ansible Community Code of Conduct.
+
+The removal process (in all cases except resignation) consists of the
+following steps:
+
+1. Another committee member creates an issue under the community-topics repository.
+2. The issue will be discussed in a community meeting.
+3. The committee members vote on the issue.
```

Comment:
Again, I don't think we should define a new procedure based on meetings which no longer will be relevant.

Reply:
Make sense. Will modify it accordingly to async process.